PayPal.xcodeproj/project.pbxproj

```diff
@@ -2573,7 +2580,7 @@
 				CLANG_ENABLE_MODULES = YES;
 				CURRENT_PROJECT_VERSION = 1;
 				DRIVERKIT_DEPLOYMENT_TARGET = 19.0;
-				EMBEDDED_CONTENT_CONTAINS_SWIFT = YES;
+                EMBEDDED_CONTENT_CONTAINS_SWIFT = YES;
```

Comment:
I have no idea why this diff shows up like this. These lines look identical in Xcode and I've even tried copying same line from main and it shows no diff from what I have.